tap_facebook_pages/streams.py

```diff
@@ -687,10 +687,10 @@ def parse_response(self, response: requests.Response) -> Iterable[dict]:
 #         # "post_video_ad_break_ad_cpm",
 #     ]
 
-class VideoInsightsStream(InsightsStream):
+class VideoInsightsLifetimeStream(InsightsStream):
```

Comment:
Should we explicitly set the period to `lifetime`? Just want to make sure we defend against FB randomly deciding to also return daily values here in the future